text/0000-default-field-values.md

```diff
@@ -0,0 +1,1760 @@
+- Feature Name: `default_field_values`
+- Start Date: 2024-08-22
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Allow `struct` definitions to provide default values for individual fields and
+thereby allowing those to be omitted from initializers. When deriving `Default`,
+the provided values will then be used. For example:
+
+```rust
+#[derive(Default)]
+struct Pet {
+    name: Option<String>, // impl Default for Pet will use Default::default() for name
+    age: i128 = 42; // impl Default for Pet will use the literal 0 for age
```

Comment:
```suggestion
    age: i128 = 42, // impl Default for Pet will use the literal 0 for age
```

typo?

Reply:
yep, carried it over from a 4 year old typo 🤦